src/components/biggive-main-menu/test/biggive-main-menu.spec.tsx

```diff
@@ -5,47 +5,245 @@ describe('biggive-main-menu', () => {
   it('renders', async () => {
     const page = await newSpecPage({
       components: [BiggiveMainMenu],
-      html: `<biggive-main-menu></biggive-main-menu>`,
+      html: `<biggive-main-menu
+                 donate-url-prefix="https://donate-here.biggive.org"
+                 blog-url-prefix="https://read-the-blog-here.biggive.org"
+                 experience-url-prefix="https://have-an-experience-here.biggive.org"
+                 is-logged-in="false"
+         ></biggive-main-menu>`,
     });
+
     expect(page.root).toEqualHtml(`
```

Comment:
I'm not sure if there's an easy way to update these expectations - I ended up deleting it and then pasting it all back in from the test failure message.